chpl/chpl-service/src/main/java/gov/healthit/chpl/realworldtesting/manager/RealWorldTestingManager.java

```diff
@@ -96,16 +98,14 @@ private ChplOneTimeTrigger startRwtUploadJob(List<RealWorldTestingUpload> rwts)
 
             List<CSVRecord> records = parser.getRecords();
             if (records.size() <= 1 && doesHeaderRowExist(records)) {
-                throw new ValidationException(
-                        "The file appears to have a header line with no other information. "
+                throw new ValidationException("The file appears to have a header line with no other information. "
                         + "Please make sure there are at least two rows in the CSV file.");
             }
 
-            //Remove the header row before processing, if it exists
+            records = removeEmptyRows(records);
```

Comment:
Should this be before the previous check? If someone uploads a file with a header and a bunch of blank lines it'd pass the previous check. Unless, if there are no rows left after removing blank and the headers it doesn't do anything? What kind of report does it generate? I think it would be reasonable for that to be a validation exception on upload as opposed to an output email thing